arangod/Scheduler/SocketTask.cpp

```diff
@@ -464,17 +333,42 @@ bool SocketTask::trySyncRead() {
 }
 
 // caller must hold the _lock
```

Comment:
I think this is old and should be removed.

Reply:
change to `must be invoked on strand`

Reply:
done